src/views/TournamentRegister.vue

```diff
@@ -131,7 +138,7 @@ const create = ref(true);
         </ul>
         <div class="self-center text-3xl">
           <input id="check" type="checkbox"/>
-          <label for="check"> J'accepte les <a :href="trules" class="text-[#63d1ff]">règles du tournoi</a></label>
+          <label for="check"> J'accepte les <a :href="`/tournament/${tournament?.id}#rules`" class="text-[#63d1ff]">règles du tournoi</a></label>
```

Comment:
is this route already exist?

Reply:
yes it is the route to tournamentDetail rules section, but it is no longer correct